tests/manifold/spherical_manifold_05.output

```diff
@@ -0,0 +1,3 @@
+
+DEAL::-0.500000 0.00000
+DEAL::0.00000 0.00000 0.500000
```

Comment:
are these the correct signs for the tangent vector? the code for the output is
```
deallog << manifold_2.get_tangent_vector (Point<2>(0.5, 0),
￼ +                                            Point<2>(1.0, 0)) << std::endl
￼ +          << manifold_3.get_tangent_vector (Point<3>(0, 0, -0.5),
￼ +                                            Point<3>(0, 0, -1.0)) << std::endl;
```
so I would have expected a positive sign in the first line and a negative sign in the second...